lib/WorkflowMain.groovy

```diff
@@ -66,7 +66,7 @@ class WorkflowMain {
         NfcoreTemplate.checkConfigProvided(workflow, log)
 
         // Check that conda channels are set-up correctly
-        if (params.enable_conda) {
+        if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
```

Comment:
what does this do? I know you posted quite a few Nextflow/CI learnings in slack to get these working, particularly for conda

Reply:
ya this is another of the nf-core changes that enables the conda declarations